generated/nimodinst/session.py

```diff
@@ -185,13 +185,13 @@ def _get_error_description(self, error_code):
         # We hand-maintain the code that calls into self._library rather than leverage code-generation
         # because niModInst_GetExtendedErrorInfo() does not properly do the IVI-dance.
         # See https://github.com/ni/nimi-python/issues/166
-        error_info_buffer_size_ctype = visatype.ViInt32()
-        error_info_ctype = None
+        error_info_buffer_size_ctype = visatype.ViInt32()  # case S170
+        error_info_ctype = None  # case C050
         error_code = self._library.niModInst_GetExtendedErrorInfo(error_info_buffer_size_ctype, error_info_ctype)
         if error_code <= 0:
             return "Failed to retrieve error description."
-        error_info_buffer_size_ctype = visatype.ViInt32(error_code)
-        error_info_ctype = ctypes.create_string_buffer(error_info_buffer_size_ctype.value)
+        error_info_buffer_size_ctype = visatype.ViInt32(error_code)  # case S180
+        error_info_ctype = (visatype.ViChar * error_info_buffer_size_ctype.value)()  # case C060
```

Comment:
comment says this is hand-maintained yet you have the `# case` comments. How come?

Reply:
I copied those pieces. I can remove the case XXXX if you want me to.

Reply:
I don't feel strongly about it, simply was confused by it.